### PR TITLE
Pypi install

### PR DESCRIPTION
**Pull request by Shimwell**
This pull request is an attempt to address issues discussed in #26 

The additional feature is the ability to pip install the code

Changes were needed to import statements, a new setup.py file and a small change to the readme.md

Distribution to PYPI test has been carried out and can easily be moved to the complete PYPI if people are happy with the PR.

Thanks for taking a look

### CODE REVIEW

**Comment by Shimwell** [comment]
Looks like the tests are failing or the Circle CI is failing in another way. 

Perhaps these test failures are related to the PIP install. I can't get and sense out of the Circle CI output but runing the tests locally it looks like there are several errors but perhaps they are not related to PIP install. The run_tests outputs ...

`FAIL: test_explode_macrobody (__main__.TestMCNPInputMethods)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "UnitTestMCNPInput.py", line 67, in test_explode_macrobody
    self.assertEqual(new_surfaces[0].surface_type,SurfaceCard.SurfaceType["PLANE_X"])
AssertionError: <SurfaceType.PLANE_X: 1> != <SurfaceType.PLANE_X: 1>

======================================================================
FAIL: test_flatten_macrobodies_with_multiple_macrobodies (__main__.TestMCNPInputMethods)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "UnitTestMCNPInput.py", line 189, in test_flatten_macrobodies_with_multiple_macrobodies
    self.assertEqual(input.surface_list[0].surface_type, SurfaceCard.SurfaceType["PLANE_X"])
AssertionError: <SurfaceType.PLANE_X: 1> != <SurfaceType.PLANE_X: 1>

======================================================================
FAIL: test_flatten_macrobodies_with_other_surfs (__main__.TestMCNPInputMethods)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "UnitTestMCNPInput.py", line 144, in test_flatten_macrobodies_with_other_surfs
    self.assertEqual(input.surface_list[0].surface_type, SurfaceCard.SurfaceType["SPHERE_GENERAL"])
AssertionError: <SurfaceType.SPHERE_GENERAL: 7> != <SurfaceType.SPHERE_GENERAL: 7>
`

**Comment by makeclean** [comment]
Ok weird, i see different failures to you. Locally I dont get your fails, maybe some cached files? I fixed some tests, but now CI fails to build numpy due to compiler age - https://github.com/numpy/numpy/issues/14182 - im going to try a different CI image

**Comment by Shimwell** [comment]
Or perhaps a different version of numpy

**Comment by makeclean** [comment]
Ok, thats got it. Ill start looking through your commits - this looks really great @shimwell thanks!

File: README.md [hunk 1]
```diff
@@ -4,15 +4,11 @@ A tool to translate common Monte Carlo geometry formats between each other.
 ## How to use
 Right now the code is petty rough; as is the install procedure.
 
-Clone the repository
-```sh
-$ git clone https://github.com/makeclean/csg2csg
+Install using pip
+```pip install csg2csg
```
Comment (makeclean) [inline README.md]: Should this not be
```
pip3 install csg2csg --user
```

**Comment by makeclean** [comment]
One other question, at what point should this be moved from makeclean/csg2csg to its own csg2csg owner, that way one could get a ghpage like - csg2csg.io - which would be nice :)

**Comment by Shimwell** [comment]
> One other question, at what point should this be moved from makeclean/csg2csg to its own csg2csg owner, that way one could get a ghpage like - csg2csg.io - which would be nice :)

**Pull request merged**